components/tools/OmeroJava/test/integration/ModelMockFactory.java

```diff
@@ -311,6 +311,18 @@ public Experiment simpleExperiment() throws ServerError {
     }
 
     /**
+     * Create a FileAnnotation with Original File.
+     * @return the FileAnnotation
+     * @throws Exception unexpected
```

Comment:
this shouldn't need to throw an exception, remove `throws` from `createOriginalFile`